GLAA.Web/Startup.cs

```diff
@@ -102,10 +102,10 @@ public void ConfigureServices(IServiceCollection services)
             services.AddTransient<IEmailSender, EmailSender>();
             services.AddTransient<IFormDefinition, LicenceApplicationFormDefinition>();
             services.AddTransient<IFieldConfiguration, FieldConfiguration>();
-            services.AddTransient<ISessionHelper, SessionHelper>();
+            services.AddScoped<ISessionHelper, SessionHelper>();
 
             // automapper
-            services.AddTransient<IMapper>(x => new AutoMapperConfig().Configure().CreateMapper());
+            services.AddSingleton<IMapper>(x => new AutoMapperConfig().Configure().CreateMapper());
```

Comment:
Was this the line which caused perf issues?

Reply:
Yes, changing this seemed to help.  Any potential issues you can see with this being a singleton now?